install-oracle.sh

```diff
@@ -204,7 +204,7 @@ BACKUP_LOG_LOCATION="${BACKUP_LOG_LOCATION:-/home/oracle/logs}"
 BACKUP_LOG_LOCATION_PARAM="^/.+$"
 
 INSTANCE_IP_ADDR="${INSTANCE_IP_ADDR}"
-INSTANCE_IP_ADDR_PARAM='^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$'
+INSTANCE_IP_ADDR_PARAM="[a-z0-9][a-z0-9\-\.]*"
 
```

Comment:
It will help if you leave maybe an inline comment with examples of what these Regexes evaluate to.

Also, if the new one bombed, do you think retaining the earlier one but commented out is a good idea?

Reply:
Added a descriptive comment.  In addition to being simpler, this new regex is a superset of the old;  it should match any 0.0.0.0 style address, with admittedly looser checks.

Reply:
Thanks